### PR TITLE
[ADD] Adding README.md file

### PR DESCRIPTION
**Pull request by ultimatecoder**
* **Description:** Adding readme file to provide the descriptive information to the user who is browsing to this repository. It will help to get better introduction of the plugin.

### CODE REVIEW

**Comment by ultimatecoder** [comment]
Any update here? Thanks!

**Comment by ultimatecoder** [comment]
@dpelle @Konfekt  Can you review this? Thanks!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+Take help from below command to have more details about installing LanguageTool.
+
+```
+:help languagetool-installation
+```
+
+### Installing vim-LanguageTool
+
+You can use any popular vim plugin installar according to your convenience.
```
Comment (Konfekt) [inline README.md]: Use any plugin manager, such as vim-plug or vundle.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+* `:help LanguageTool`
+
+  To get more details on various commands and configuration information.
+
+
+## Installation instructions
+
+To use this plugin, you need to install the Java LanguageTool program.
+
+### Download the LanguageTool
```
Comment (Konfekt) [inline README.md]: Download LanguageTool

File: README.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+:help languagetool-installation
+```
+
+### Installing vim-LanguageTool
+
+You can use any popular vim plugin installar according to your convenience.
+
+#### [Vundle](https://github.com/VundleVim/Vundle.vim)
+
+Add below text to your .vimrc file and save:
```
Comment (Konfekt) [inline README.md]: Add the text below to your `.vimrc` file and save it

**Comment by Konfekt** [comment]
Thanks! Seems fine to me, pointed out minor corrections. Everything hinges on @dpelle though.

**Comment by dpelle** [comment]
Thanks. I've merged the pull request.  I'll apply the corrections suggested by Konfekt.

**Comment by ultimatecoder** [comment]
Thanks for merging this. I will try to update them if I will get time. Thanks :)

**Pull request merged**